tests/http/redirects.test.js

```diff
@@ -0,0 +1,57 @@
+var server = require('../../server');
```

Comment:
It's only have the picture though, as this is passed off to react router and tested separatly.